db/migrate/20220321235014_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
```

Comment:
- [ ] The same thing here, please add the `comments_counter` and the `likes_counter` columns. :+1:

Reply:
thanks 👍🏽 I forgot haha